packages/extension-floating-menu/package.json

```diff
@@ -29,16 +29,15 @@
     "dist"
   ],
   "devDependencies": {
+    "@floating-ui/dom": "^1.6.0",
     "@tiptap/core": "^2.5.7",
     "@tiptap/pm": "^2.5.7"
   },
   "peerDependencies": {
+    "@floating-ui/dom": "^1.6.0",
```

Comment:
This is a dep not peer dep though? You call its functions so it has to be included, no?

Reply:
This is a library that consumers of Tiptap might be using themselves in their apps for other purposes, so in a way, I think it makes sense to be a peer dependency. However, maybe a better solution would be `@tiptap/floating`, similar to `@tiptap/pm`?

Reply:
It seems I had a misunderstanding of what peer deps do. My mistake it does make sense to have it as a peer dep. Though the specified range should be much wider probably just `^1` or whenever the functionality that we use in the package was introduced. This would conflict less with library consumer's versions.